src/main/kotlin/no/nav/syfo/application/metrics/MetricRegister.kt

```diff
@@ -1,6 +1,5 @@
 package no.nav.syfo.application.metrics
 
-import io.prometheus.client.Counter
 import io.prometheus.client.Histogram
 
 const val METRICS_NS = ""
```

Comment:
Vi kunne kanskje hatt en counter for hver melding den skriver til kø så vi ser hvis den slutter å gjøre noe?